### PR TITLE
[OSPK8-420] Use jumbo frames for br-osp + introduce test scnarios

### PR DESCRIPTION
**Pull request by stuggi**
Use jumbo frames for libvirt ospnetwork, br-osp and the underlying interface
to test common scenario where some of the osp networks use jumbo frames.

Also:
- changes to use 4 network files for ipv4, ipv4_subnet, ipv6 and ipv6_subnet.
  The scenario config files reference these network configurations in osp.networks.
  This allows to change in a single place the networks configuration for all
  instead of edit all of them when there is a change needed.

- removes duplicate environment files for e.g. 16.2 and train  + 17.0 and walaby
  by using links. Only keep things separate for 16.2/train where upstream is
  different then downstream (roles file)

- adds new configs
  - 16.2_tlse_subnet
  - 16.2_hci_tlse_subnet
  - 16.2_novacontrol_tlse_subnet
  - 16.2_novacontrol_hci_tlse_subnet
  - 16.2_ipv6_hci_subnet
  - 16.2_ipv6_novacontrol_hci_tlse_subnet
  - 16.2_fencing_subnet
  - train_subnet
  - train_ipv6_subnet
  - wallaby_subnet
  - wallaby_ipv6_subnet

Depends-On: https://github.com/openstack-k8s-operators/osp-director-operator/pull/474

### CODE REVIEW

File: ansible/templates/osp/netconfig/osnetconfig.yaml.j2 [hunk 1]
```diff
@@ -11,19 +11,27 @@ spec:
...
-            description: Linux bridge with {{ nodeConfPolicy.interface }} as a port
-            name: {{ nodeConfPolicy.bridge_name }}
-            state: up
+              - name: {{ ifcfg.interface }}
+            description: Linux bridge with {{ ifcfg.interface }} as a port
+            name: {{ ifcfg.name }}
             type: linux-bridge
-{% if nodeConfPolicy.mtu is defined %}
-            mtu: {{ nodeConfPolicy.mtu }}
+{% elif ifcfg.type >=  "ethernet" %}
```
Comment (abays) [inline ansible/templates/osp/netconfig/osnetconfig.yaml.j2:25]: Is this `>=` supposed to be here

Comment (stuggi) [reply ansible/templates/osp/netconfig/osnetconfig.yaml.j2]: no, that should be ==, thanks!

**Review by abays — APPROVED** [review]
/lgtm

**Comment by stuggi** [comment]
Job 603 passed for AI+dev-tools, merging the PR

**Pull request merged**